controller/accounts.js

```diff
@@ -54,10 +117,48 @@ router.get('/averageage', function(req, res){
   }
 });
 
+//post passes start and end dates to render fresh graph
+router.post('/averageage', function(req, res){
+  if(!sess || !sess.username){
+    res.redirect('/');
+  } else {
+    console.log(req.body.start);
+    console.log(req.body.end);
+    let start = new Date(req.body.start).toISOString().slice(0, 10);
+    let end = new Date(req.body.end).toISOString().slice(0, 10);
+    console.log(start,end);
+    let averageage = kujiraDataMiner.averageAge(message, start, end);
```

Comment:
maybe some error handling here, in case kujiraDataMiner returns bad data, with below calls to fs & render being integrated into that error handling (ie only being called if averageage data is good)